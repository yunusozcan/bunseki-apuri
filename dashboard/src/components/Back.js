import React from 'react';
import back from '../images/back.svg';
import '../stylesheets/components/Back.css';
import {Link} from "react-router-dom";

function Back() {
    return (
        <Link to={"/"} className="App-back App-button">
            <img src={back} className="App-back-image" alt="back"/>
        </Link>
    );
}

export default Back;
