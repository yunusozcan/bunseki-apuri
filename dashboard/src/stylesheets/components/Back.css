.App-back {
    flex-grow: 1;
    text-align: center;
    min-width: 30px;
    max-width: 50px;

}

.App-back-image {
    width: 10px;
    height: 10px;
    padding-top: 17px;
}
