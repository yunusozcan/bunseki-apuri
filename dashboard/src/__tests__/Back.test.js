import React from 'react';
import Back from '../components/Back';
import renderer from 'react-test-renderer';
import {BrowserRouter as Router} from "react-router-dom";

it('renders correctly', () => {
    const tree = renderer
        .create(
            <Router>
                <Back/>
            </Router>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
