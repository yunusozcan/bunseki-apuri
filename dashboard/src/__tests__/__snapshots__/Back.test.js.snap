// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="App-back App-button"
  href="/"
  onClick={[Function]}
>
  <img
    alt="back"
    className="App-back-image"
    src="back.svg"
  />
</a>
`;
